docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md

```diff
@@ -0,0 +1,347 @@
+---
+description: Provides guidance on how to configure the auto-update and repair settings of a Windows App.
+external help file: Microsoft.Windows.Appx.PackageManager.Commands.dll-help.xml
+Module Name: Appx
+ms.date: 06/07/2021
+online version: https://docs.microsoft.com/powershell/module/appx/reset-appxpackage?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Set-AppxPackageAutoUpdateSettings
+---
+
+# Set-AppxPackageAutoUpdateSettings
+
+## SYNOPSIS
+Available in the Windows Insider Preview Builds of windows 10, is the `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet. The `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet provides access to configure a specific Windows App's Auto Update and Repair settings. Including pausing update checks.
+
+## SYNTAX
+
+### SetAutoUpdateOptionsSet (Default)
+```
+Set-AppxPackageAutoUpdateSettings [-PackageFamilyName] <String> -AppInstallerUri <String>
+ [-UpdateUris <String[]>] [-RepairUris <String[]>] [-OptionalPackages <String[]>]
+ [-DependencyPackages <String[]>] [-EnableAutomaticBackgroundTask] [-ForceUpdateFromAnyVersion]
+ [-DisableAutoRepairs] [-CheckOnLaunch] [-ShowPrompt] [-UpdateBlocksActivation] [-UseSystemPolicySource]
+ [-HoursBetweenUpdateChecks <UInt32>] [-Version <String>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### PauseAutoUpdateOptionsSet
+```
+Set-AppxPackageAutoUpdateSettings [-PackageFamilyName] <String> [-PauseUpdates] [-UseSystemPolicySource]
+ -HoursToPause <UInt32> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Set-AppxPackageAutoUpdateSettings** cmdlet provides access to configure a specific Windows App's Auto Update and Repair seetings. 
+
+## EXAMPLES
+
+### Example 1: Update the Auto-Update settings for an App
+```
+PS C:\> Set-AppxPackageAutoUpdateSettings -AppInstallerUri https://website.com/PackageName.appinstaller -PackageFamilyName PackageName_8h66172c634n0 -CheckOnLaunch -ForceUpdateFromAnyVersion -HoursBetweenUpdateChecks 2 -ShowPrompt -UpdateUris file://ComputerName/Share/PackageName_x64.appinstaller
+```
+
+This cmdlet will update the Auto-Update settings of the *PackageName_8h66172c634n0* Windows App to target an AppInstaller file on a network accessible file share every two hours, displaying a prompt to the user. Allowing for the Windows App to update to any version (higher or lower) despite the version of the installed Windows App.
```

Comment:
```suggestion
This cmdlet will update the Auto Update settings of the *PackageName_8h66172c634n0* Windows App to target an AppInstaller file on a network accessible file share every two hours, displaying a prompt to the user. Allowing for the Windows App to update to any version (higher or lower) despite the version of the installed Windows App.
```

Reply:
Let´s use the same syntax through the article for Auto Update and Repair settings